cmd/mothership/mothership/start/scheduler.go

```diff
@@ -56,6 +58,15 @@ func startScheduler(ctx context.Context, o *Options, schedulerCfg *config.Config
 		Run(ctx)
 }
 
+func getReconcilers(cfg *config.Config) []string {
+	reconcilerList := make([]string, 0, len(cfg.Scheduler.Reconcilers)+1)
+	for reconciler := range cfg.Scheduler.Reconcilers {
+		formattedReconciler := strings.Replace(reconciler, "-", "_", -1)
```

Comment:
Don't think this is the right place for such logic: if the metrics canno deal with the names as they are, the conversion of these names should happen inside of the metrics struct.